### PR TITLE
BGDIINF_SB-1981: updated spec for asset upload MD5 integrity check

### PR DESCRIPTION
**Pull request by ltshb**
Spec deployed here: https://service-stac.dev.bgdi.ch/api/stac/static/spec/v0.9/apitransactional.html#tag/Asset-Upload-Management

### CODE REVIEW

File: spec/static/spec/v0.9/openapitransactional.yaml [hunk 1]
```diff
@@ -785,7 +793,12 @@ paths:
       - Asset Upload Management
       summary: Create a new Asset's multipart upload
       description: |
-        Create a new Asset's multipart upload.
+        Create Asset's multipart upload.
+
+        A file part must be at least 5 MB except for the last one and at most 5 GB, otherwise the complete operation will fail.
+
+        *Note: in order to provide integrity check during the upload, the base64-encoded 128-bit MD5 digest of each part must
```
Comment (hansmannj) [inline spec/static/spec/v0.9/openapitransactional.yaml:800]: must + a _be_ computed (picky, I know, sorry :wink:)

File: spec/transaction/paths.yaml [hunk 2]
```diff
@@ -474,7 +474,12 @@ paths:
         - Asset Upload Management
       summary: Create a new Asset's multipart upload
       description: |
-        Create a new Asset's multipart upload.
+        Create Asset's multipart upload.
+
+        A file part must be at least 5 MB except for the last one and at most 5 GB, otherwise the complete operation will fail.
+
+        *Note: in order to provide integrity check during the upload, the base64-encoded 128-bit MD5 digest of each part must
```
Comment (hansmannj) [inline spec/transaction/paths.yaml:481]: dito (must _be_ computed)

**Review by hansmannj — APPROVED** [review]
:1st_place_medal: Thanks!

File: spec/static/spec/v0.9/openapitransactional.yaml (hunk 1 above)
Comment (ltshb) [reply spec/static/spec/v0.9/openapitransactional.yaml:800]: Not picky but right :-) I'll fix it in next PR though.

**Pull request merged**